build.sbt

```diff
@@ -50,8 +50,12 @@ scalaVersion := "2.12.15"
 // keep this as low as possible to avoid running into binary incompatibility such as https://github.com/sbt/sbt/issues/5049
 pluginCrossBuild / sbtVersion := "1.2.1"
 
-// first release that can build 2.13 (as it bring a Zinc version with a compiler-bridge published for 2.13)
-scriptedSbt := "1.2.7"
+scriptedSbt := {
+  if (System.getProperty("java.specification.version").toDouble < 17)
+    "1.2.7" // first release that can build 2.13 (as it bring a Zinc version with a compiler-bridge published for 2.13)
+  else
+    "1.5.5" // first release that supports JDK17
```

Comment:
https://github.com/scala/docs.scala-lang/pull/2241/files#diff-c3d3de6eac2eb60c40c2ff73d17ba59d2442a65959b0696592bd3829851ecce2R78

Reply:
interestingly, this bump revealed that a minor feature works with sbt 1.2.7 (<1.3) but not on sbt 1.5.5 (>1.3), see https://github.com/scalacenter/sbt-scalafix/pull/274/commits/30d8d4d950a2e8741d7c89d87d7bb3a4b7b604b1